episodes/chap8_own_project.md

```diff
@@ -47,9 +48,25 @@ algorithm. Now, you can add privacy filters to your algorithm.
   exceptions](https://docs.vantage6.ai/en/main/function-docs/_autosummary/vantage6.algorithm.tools.exceptions.html)
   to return an error message.
 
-Hint: the [contingency table algorithm](https://github.com/vantage6/v6-crosstab-py)
+:::::::::::::::: hint
+
+The [contingency table algorithm](https://github.com/vantage6/v6-crosstab-py)
```

Comment:
Could you add the line numbers or permalinks or something to indicate which part is actually a privacy filter?

Reply:
Done